### ISSUE TITLE
Can't build in visual studio 2012

### ISSUE THREAD

**Issue by Shaderboy** [obviousjim/ofxOculusDK2#13]
Hi, I'm using visual studio 2012 and have been trying to set up both and empty project and the example project with no luck. Here's what I've tried so far:
1. Building the example project: This gives a bunch of errors in intellisense and won't compile.
2. Creating a new project with the generator and including the addon: The intellisense errors were gone here, but when compiling it complains about not being able to find Unix related files. Obviously the generator is telling it to include the osx specific library.
3. Creating a new project with the generator but NOT including the addon, then manually including \src, \libs\LibOVR\Src and \libs\LibOVR\Include in additional include directories and adding \libs\LibOVR\Lib\x64\VS2012 and \ofxOculusDK2Lib to the linker's additional library directories (I also tried including \libs). This solves all the "cannot find .h" errors, but creates a bunch of linker errors.

The errors are 'unresolved external symbol' errors for the ofxOculusDK2 class's functions.

Does the addon definitely work with visual studio 2012, and if so what am I missing here?

**Comment by jamesalliban** [comment]
Same problem here. Did you manage to solve this?

**Comment by Shaderboy** [comment]
No, sadly I never did. I switched over to Unity for that project instead and the oculus plugin for it was really easy to set up and use.

**Comment by jamesalliban** [comment]
OK thanks for your response.
